### DISCUSSION TITLE
Porsche Macan 2023

### DISCUSSION THREAD

**Discussion by jeromebt** [Mr-MIBonk/M.I.B._More-Incredible-Bash#262] [General]
Hello, would your update work to activate Android Auto on a Porsche Macan 2023 PCM 5.1
Version Software MH2p_US_PO416_P270?

Thank you
Jerome

**Comment by harman-f** [comment]
This solution is vor MHI2 based units only.

MH2p is newer, look for toolbox on mibsolution.one

**Reply by jeromebt** [reply]
Does the solution work withh USB drive, or you need the SD card ?

**Comment by harman-f** [comment]
SD card as far as I know + D-Link.

Check mibsolution.one

**Comment by jeromebt** [comment]
I don't really understand how to do it....

Jerome
________________________________
